visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -343,6 +419,30 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
         }
     }
 
+    private readonly featureChanged = (ev: React.FormEvent<HTMLInputElement>, option: IDropdownOption): void => {
+        const featureKey = option.key.toString();
+        if (this.state.featureKey !== featureKey) {
+            this.props.featureBinPickerProps.selectedBinIndex = this.props.dashboardContext.modelMetadata.featureNames.indexOf(featureKey);
+            this.setState({featureKey: featureKey, disparityArray: undefined});
+        }
+    }
+
+    private readonly accuracyChanged = (ev: React.FormEvent<HTMLInputElement>, option: IDropdownOption): void => {
+        const accuracyKey = option.key.toString();
+        if (this.state.accuracyKey !== accuracyKey) {
+            this.props.accuracyPickerProps.selectedAccuracyKey = accuracyKey;
+            this.setState({accuracyKey: accuracyKey, disparityArray: undefined});
```

Comment:
changing accuracyKey should clear the accuracyArray, to make sure the correct value is fetched. Right now I think if either is undefined we fetch everything from scratch, but in a better optimized way it should probably only fetch the accuracyArray if accuracy changes. Above, if the feature changes, then both accuracy and disparity should be cleared and refetched